Duckling/Time/EN/Rules.hs

```diff
@@ -1118,6 +1118,30 @@ ruleIntervalFromDDDDMonth = Rule
       _ -> Nothing
   }
 
+ruleIntervalFromDDDDOfMonth :: Rule
+ruleIntervalFromDDDDOfMonth = Rule
+  { name = "from the <day-of-month> (ordinal or number) to the <day-of-month> (ordinal or number) of <named-month> (interval)"
+  , pattern =
+    [ regex "from( the)?"
+    , Predicate isDOMValue
+    , regex "\\-|to( the)?|th?ru|through|(un)?til(l)?"
+    , Predicate isDOMValue
+    , regex "of"
+    , Predicate isAMonth
+    ]
+  , prod = \tokens -> case tokens of
+      (_:
+       token1:
+       _:
+       token2:
+       Token Time td:
```

Comment:
```suggestion
        _:
       Token Time td:
```

Reply:
That was it. Thank you @patapizza for spotting that. Makes a lot more sense now 👍